modules/aleph/includes/aleph.loan.inc

```diff
@@ -38,3 +38,33 @@ function aleph_loan_list() {
 
   return $results;
 }
+
+/**
+ * Implements provider loan, renew.
+ *
+ * @param object $account
+ *    Drupal user object to renew loan for.
+ * @param array $loan_ids
+ *    The loan IDs.
+ *
+ * @return array Result of renewals.
+ *    Result of renewals.
+ */
+function aleph_loan_renew($account, array $loan_ids) {
+  $patron = aleph_get_patron_by_global_account($account);
```

Comment:
Does seem a bit wird to pass an account object to a function whos name indicates that it is going to be using a global account ...